sdk/nodejs/automation/index.ts

```diff
@@ -23,3 +23,4 @@ export * from "./workspace";
 export * from "./events";
 export * from "./remoteStack";
 export * from "./remoteWorkspace";
+export * from "./tag";
```

Comment:
This exports a single type, which is used in a public API (workspace.listTags)

https://github.com/pulumi/pulumi/blob/master/sdk/nodejs/automation/tag.ts